
2022-04-01 23:26:20,060	INFO services.py:1412 -- View the Ray dashboard at [32m[1mhttp://127.0.0.1:8265
[36m(ServeController pid=32025)[39m 2022-04-01 23:26:22,956	INFO checkpoint_path.py:16 -- Using RayInternalKVStore for controller checkpoint and recovery.
[36m(ServeController pid=32025)[39m 2022-04-01 23:26:23,066	INFO http_state.py:98 -- Starting HTTP proxy with name 'SERVE_CONTROLLER_ACTOR:XEQVhZ:SERVE_PROXY_ACTOR-node:127.0.0.1-0' on node 'node:127.0.0.1-0' listening on '127.0.0.1:8000'
2022-04-01 23:26:23,310	INFO api.py:521 -- Started Serve instance in namespace 'serve'.
[36m(HTTPProxyActor pid=32023)[39m INFO:     Started server process [32023]
2022-04-01 23:26:23,335	INFO simple_q.py:154 -- In multi-agent mode, policies will be optimized sequentially by the multi-GPU optimizer. Consider setting `simple_optimizer=True` if this doesn't work for you.
2022-04-01 23:26:23,335	INFO trainer.py:779 -- Current log_level is WARN. For more information, set 'log_level': 'INFO' / 'DEBUG' or use the -v and -vv flags.
2022-04-01 23:26:23,391	WARNING util.py:55 -- Install gputil for GPU system monitoring.
Traceback (most recent call last):
  File "/Users/liwenyu/Documents/GitHub/RTDM/loadmodel.py", line 40, in <module>
    trainer.restore(trained_model)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/ray/tune/trainable.py", line 490, in restore
    self.load_checkpoint(checkpoint_path)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/ray/rllib/agents/trainer.py", line 1861, in load_checkpoint
    self.__setstate__(extra_data)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/ray/rllib/agents/trainer.py", line 2509, in __setstate__
    self.workers.local_worker().restore(state["worker"])
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/ray/rllib/evaluation/rollout_worker.py", line 1333, in restore
    objs = pickle.loads(int(objs))
ValueError: invalid literal for int() with base 10: b'\x80\x05\x95vJ\x00\x00\x00\x00\x00\x00}\x94(\x8c\x07filters\x94}\x94\x8c\x0edefault_policy\x94\x8c\x16ray.rllib.utils.filter\x94\x8c\x08NoFilter\x94\x93\x94)\x81\x94s\x8c\x05state\x94}\x94h\x03}\x94
Traceback (most recent call last):
  File "/Users/liwenyu/Documents/GitHub/RTDM/loadmodel.py", line 40, in <module>
    trainer.restore(trained_model)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/ray/tune/trainable.py", line 490, in restore
    self.load_checkpoint(checkpoint_path)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/ray/rllib/agents/trainer.py", line 1861, in load_checkpoint
    self.__setstate__(extra_data)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/ray/rllib/agents/trainer.py", line 2509, in __setstate__
    self.workers.local_worker().restore(state["worker"])
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/ray/rllib/evaluation/rollout_worker.py", line 1333, in restore
    objs = pickle.loads(int(objs))
ValueError: invalid literal for int() with base 10: b'\x80\x05\x95vJ\x00\x00\x00\x00\x00\x00}\x94(\x8c\x07filters\x94}\x94\x8c\x0edefault_policy\x94\x8c\x16ray.rllib.utils.filter\x94\x8c\x08NoFilter\x94\x93\x94)\x81\x94s\x8c\x05state\x94}\x94h\x03}\x94